examples/md-flexible/src/ParallelVtkWriter.cpp

```diff
@@ -231,7 +231,7 @@ void ParallelVtkWriter::tryCreateSessionAndDataFolders(const std::string &name,
 
 void ParallelVtkWriter::createPvtuFile(const int &currentIteration) {
   std::ostringstream filename;
-  filename << _sessionFolderPath << _sessionName << "_particles_" << std::setfill('0')
+  filename << _sessionFolderPath << _sessionName << "_" << std::setfill('0')
```

Comment:
why do you want to remove this? Seems like a useful part of the name.
(Also applies to the other locations where you removed similar things)

Reply:
I think it is redundant and introduces additional maintainance effort. 
The file type already provides a good hint about the content of the file, which is why I consider this information redundant.
If for some reason someone wants to update the information to say "particles_and_other_stuff" this not only the ParallelVtkWriter requires an update but also the MDFlexConfig for loading checkpoints properly. I fell into that trap :)